.vscode/settings.json

```diff
@@ -8,7 +8,8 @@
   "editor.tabSize": 2,
 
   "files.associations": {
-    "*.anm2": "xml"
+    "*.anm2": "xml",
+    ".anm2": "xml"
```

Comment:
is this change intentional? it looks like a bug

Reply:
I think that my vscode changed that for some reason. That change can be ignored.